src/index.html

```diff
@@ -17,6 +17,22 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href = "#" class="logo">
+        <img src="images/logo.png" alt="Moyo Logo">
+      </a>
+      <nav>
+        <ul>
+          <li><a href="#" class ="is-active">Apple</a></li>
```

Comment:
There is an extra space around the `=` in the `class` attribute. It should be `class="is-active"` for consistency.